neon-animated-pages.html

```diff
@@ -56,6 +56,8 @@
 
     is: 'neon-animated-pages',
 
+    created: function () { console.log("created"); },
```

Comment:
Ooops yes I must be sleeping! Was doing some checks and left that in by accident.